pkg/cmd/codespace/list.go

```diff
@@ -34,11 +35,21 @@ func newListCmd(app *App) *cobra.Command {
 		`),
 		Aliases: []string{"ls"},
 		Args:    noArgsConstraint,
-		RunE: func(cmd *cobra.Command, args []string) error {
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			err := cmdutil.MutuallyExclusive("using `--org` or `--user` with `--repo` is not allowed", opts.repo != "", opts.orgName != "" || opts.userName != "")
+			if err != nil {
+				return err
+			}
+			err = cmdutil.MutuallyExclusive("using `--web` with `--org` or `--user` is not supported, please use with `--repo` instead", opts.useWeb, opts.orgName != "" || opts.userName != "")
+			if err != nil {
+				return err
+			}
```

Comment:
Nit: break these long lines up, use the more idiomatic `if err := ...; err != nil` technique, and use the same error format for each:

```suggestion
            if err := cmdutil.MutuallyExclusive(
                "using `--org` or `--user` with `--repo` is not allowed", 
                opts.repo != "",
                opts.orgName != "" || opts.userName != "",
            ); err != nil {
                return err
            }
            
            if err := cmdutil.MutuallyExclusive(
                "using `--org` or `--user` with `--web` is not allowed, only `--repo` is allowed", 
                opts.useWeb, 
                opts.orgName != "" || opts.userName != "",
            ); err != nil {
                return err
            }
```

Reply:
Understood.
fixed in [ce9ce98](https://github.com/cli/cli/pull/7288/commits/ce9ce984ba443f6421fdd077b3ba3e72cd8fff57)